### ISSUE TITLE
Disabled items shouldn't fail color-contrast rule

### ISSUE THREAD

**Issue by Develliot** [dequelabs/axe-core#2964]
Product: axe-core

Expectation: 
button that has disabled or aria-disabled shouldn't fail color-contrast rule

Actual: 
disabled button <Button disabled>blah</Button> can fail colour contrast test when greyed out

Motivation: 
https://www.w3.org/TR/WCAG21/#contrast-minimum
as the item is no longer interactive it no longer needs to follow colour contrast rules but does need to look visually different from interactive buttons, greying out disabled buttons is a standard web paradigm.  

<pre><code>
"axe-core": "4.2.1"
Browser and Assistive Technology versions
</code></pre>

**Comment by WilcoFiers** [comment]
Axe-core excludes disabled buttons from the color-contrast rule. If you've found an example where it doesn't, that is a bug. Can you provide some information on how to reproduce the issue?

**Comment by WilcoFiers** [comment]
@Develliot can you please provide some details on how to reproduce the issue?

**Comment by Develliot** [comment]
I use axe core as a dependency for cypress-axe, so that might be causing the issue. I think this ticket can be closed as I don't have a good way to demonstrate it.

**Comment by WilcoFiers** [comment]
Alright, closing the issue then.
